src/detector.js

```diff
@@ -119,6 +119,9 @@ class PhishingDetector {
 
 }
 
+PhishingDetector.processDomainList = processDomainList
+PhishingDetector.domainToParts = domainToParts
+PhishingDetector.domainPartsToDomain = domainPartsToDomain
```

Comment:
Does this impact runtime security? As in there being more areas in code where changes to the lists could be injected.